src/controller/OperationalCredentialsDelegate.h

```diff
@@ -81,6 +89,13 @@ class DLL_EXPORT OperationalCredentialsDelegate
      */
     virtual void SetFabricIdForNextNOCRequest(FabricId fabricId) {}
 
+    /**
+     *   This function sets the CAT values for which the next NOC Chain should be generated. This API is
+     *   not required to be implemented if the delegate implementation has other mechanisms to find the
```

Comment:
Then what's the point of this API?  This seems like an implementation detail leaking into interface.

Reply:
This API is intended to make the method accessible in future SDK updates where the exact derived type is unknown.

Reply:
But who is providing these CAT values that doesn't know what the operational credentials issuing mechanism is?

Reply:
These CAT values can be supplied by either the JF Administrator Server or the JFA demo app.

Reply:
All the changes in src/controller have been reverted.